### DISCUSSION TITLE
自定义顶栏的搜索框又窄又高

### DISCUSSION THREAD

**Discussion by horizon98** [the1812/Bilibili-Evolved#2710] [功能建议 / Ideas]
如下图，图1开启了“自定义顶栏”功能，图2未开启。开启后搜索框很窄，而且看起来比较高，总之有点违和的感觉。

![1](https://user-images.githubusercontent.com/48114360/145823479-05f28ba4-2025-4a56-a6ef-0f1a2fc50293.png)

![2](https://user-images.githubusercontent.com/48114360/145823491-21e9aa79-42f4-43ff-b898-1d24d0a9a82d.png)

**Comment by Geeyun-JY3** [comment]
分享个技巧啊，和开发者是否接受你这个建议无关：
安装 `快捷键扩展` 后默认可以通过 <kbd>/</kbd> 调出一个大搜索栏：
![image](https://user-images.githubusercontent.com/49396421/145835160-d794a3a2-d62d-42ec-81dc-3ccc72e827cc.png)

**Reply by horizon98** [reply]
> 分享个技巧啊，和开发者是否接受你这个建议无关： 安装 `快捷键扩展` 后默认可以通过 / 调出一个大搜索栏： ![image](https://user-images.githubusercontent.com/49396421/145835160-d794a3a2-d62d-42ec-81dc-3ccc72e827cc.png)

感谢分享！学到了😀
